app/src/components/Graph.js

```diff
@@ -133,13 +134,24 @@ const Graph = ({
       );
     };
 
+    // when new entries are added, or the probability of the selected outcome changes
     if (entries.length >= data.length || currentProbabilityChanged()) {
+      const midValue =
+        (parseFloat(upperBound) - parseFloat(lowerBound)) / 2 +
+        parseFloat(lowerBound);
       const newData = [
+        {
+          outcomesProbability: [midValue],
+          date: moment(created).valueOf(),
+          // First entry in `entries` comes with index 0 and we add this one also
```

Comment:
Added this comment here. We set the init values as index 0 and first entry also has index 0. Shouldn't be critical because we aren't currently using this index but can be in future if we start to use it